cookbook/security/impersonating_user.rst

```diff
@@ -86,6 +86,24 @@ to show a link to exit impersonation:
             </a>
         <?php endif ?>
 
+In some cases you may need to get the object that represents the impersonating
+user rather than the impersonated user. Use the following snippet to iterate
+over user's roles until you get the ``SwitchUserRole`` related to the
+impersonating user::
+
+    use Symfony\Component\Security\Core\Role\SwitchUserRole;
+
+    $securityContext = $this->get('security.context');
```

Comment:
this needs to be 'security.authorization_checker' since 'security.context' is deprecated